src/lib/bootloader/grub2base.rb

```diff
@@ -120,6 +120,9 @@ def propose
 
       propose_serial
 
+      # we proposing new config and we do not know sections yet, so propose empty
+      @sections.default = ""
```

Comment:
I still can't quite put my finger on it. Here we're changing `sections.default ` from "unknown" to "empty" so we should do the same to `sections.all`. Either here or in the constructor.